cloudinit/sources/tests/test_oracle.py

```diff
@@ -412,42 +408,102 @@ def _is_netfail_master(iface):
         self.assertEqual(expected_cfg, netcfg)
 
 
+def _mock_v2_urls(httpretty):
+    def instance_callback(request, uri, response_headers):
+        assert request.headers.get("Authorization") == "Bearer Oracle"
+        return [200, {}, OPC_V2_METADATA]
```

Comment:
It looks like the package build in xenial is failing during the tests: https://travis-ci.com/github/canonical/cloud-init/jobs/371325182

I believe this is because the version of httpretty (0.8.6) in xenial does not include https://github.com/gabrielfalcao/HTTPretty/commit/bdb7ee7c1fd10838713bbc5851fa42a598295f53 (which was first included in 0.8.11). So I think you'll need to add some default response header values to avoid these tracebacks.

(This isn't caught by the "xenial" unittests in Travis because we don't have HTTPretty pinned correctly; I'll be submitting a PR to address that shortly.)

Reply:
https://github.com/canonical/cloud-init/pull/531 is that PR.